### PR TITLE
Initialize Release Notes for 1.1

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Issue #148.

- Renamed 'history' to 'Release Notes'.
- Started section on Scoped Pages.
- Reference to the maps chart type.
- Update Reference Manual home page.
- Make sure REST has a Reference Manual home page.

### CODE REVIEW

File: docs/manuals/reference.md [hunk 1]
```diff
@@ -6,5 +6,10 @@ The API reference for Taipy is therefore split into several sections.
 
    - [`taipy`](reference/pkg_taipy.md): The main package of Taipy.
 
+   - [`taipy.core`](reference/pkg_taipy.core/index.md): The package providing .
```
Comment (FredLL-Avaiga) [inline docs/manuals/reference.md]: providing **<missing end of line>**.

File: docs/relnotes.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+     See this [example](manuals/gui/viselements/charts/others.md#plotting-on-a-map) for details.
+   - The [`input`](manuals/gui/viselements/input.md) and [`slider`](manuals/gui/viselements/slider.md)
+     controls have a new _change_delay_ property that lets you tune how fast you want to propagate
+     changes.<br/>
+     This allows for a better user experience.
+   - The [`input`](manuals/gui/viselements/input.md) control has a new _password_ property that, if True,
+     obscures the user input.
+   - The [`input`](manuals/gui/viselements/input.md), [`number`](manuals/gui/viselements/number.md) and
+     [`selector`](manuals/gui/viselements/selector.md) controls have a new _label_ property that lets you
+    display a label next to the control.
```
Comment (FredLL-Avaiga) [inline docs/relnotes.md]: it's not next it's in :-)

File: docs/relnotes.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+   - Performance improvements when reading and writing entities.
+
+#### Significant bug fixes
+
+**`taipy-gui`**<br/>1.1.0
+
+   - Concurrency issues were fixed.
+   - Taipy supports HTTPS via reverse proxies.<br/>
+     See [issue](https://github.com/Avaiga/taipy-gui/issues/263).
+   - Modifying a variable multiple times in _on_change()_  is now supported.<br/>
```
Comment (FredLL-Avaiga) [inline docs/relnotes.md]: this bug was never public 
In 1.0, we could have an infinite loop by modifying a variable in the on_change function for that variable
In 1.1, we prevented the possible infinite loop

**Review by FredLL-Avaiga — APPROVED** [review]
A few clarifications

**Review by jrobinAV — APPROVED** [review]

File: docs/manuals/reference.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/reference.md]: Fixed.

File: docs/relnotes.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/relnotes.md]: Fixed

File: docs/relnotes.md (hunk 3 above)
Comment (FabienLelaquais) [reply docs/relnotes.md]: Moved to Improvements.

File: docs/relnotes.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+[`taipy-core` 1.1](https://pypi.org/project/taipy-core/1.1.0/) and
+[`taipy-rest` 1.1](https://pypi.org/project/taipy-rest/1.1.0/) packages.
+
+
+#### Improvements
+
+**`taipy-gui`**<br/>1.1.0
+
+   - The `State^` instance can be initialized in a user-defined function. See the _on_init_
+     attribute of the `Gui^` class for more details.
```
Comment (FredLL-Avaiga) [inline docs/relnotes.md:25]: attribute => method ?
can't we write Gui.on_init^ ?

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/relnotes.md (hunk 4 above)
Comment (FabienLelaquais) [reply docs/relnotes.md:25]: Unfortunately no.
_on_init()_  has no specific entry where an anchor would be created.

We simply document this attribute in the body of the class description.

**Pull request merged**